docs/HowToFAQ/Errors-Resolution.md

```diff
@@ -652,9 +760,9 @@ Hence, indications of non-terminating loops, that is, `decreases *`, are not per
 
 This does mean that the specifier has to do the work of designing a valid terminating condition and proving it.
 
-<!-- 2 instances -->
+<!-- 3 instances -->
```

Comment:
What are these lines doing?

Reply:
They remind me that the same error message is produced by multiple spots in the code -- so I should make sure that the description is appropriately general, and perhaps add multiple examples. Keep in mind that these examples all are converted into lit scenarios that feed into the coverage measurements. When browsing the coverage results, if I see a error message that is uncovered, I try to determine whether I'm just missing a test or whether it is defensive programming that is (currently) unreachable.

Reply:
Cool! You've got some great automation on all of this. Another +1 for a bit more documentation on it. :)